src/rooms/entities/room.entity.ts

```diff
@@ -0,0 +1,29 @@
+import { Doctor } from 'src/doctors/entities/doctor.entity';
+import { Specialization } from 'src/specializations/entities/specialization.entity';
+import { User } from 'src/users/entities/user.entity';
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  OneToMany,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity({ name: 'rooms' })
+export class Room {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column()
+  name: string;
+
+  @OneToMany(() => Doctor, (doctor) => doctor.rooms)
+  @Column('character varying', { array: true })
+  doctors: Doctor[];
```

Comment:
from the task requirements i think there can be only one doctor in room, so 1 doctor can be assigned to mny rooms